internal/operators/ocs/validations.go

```diff
@@ -236,7 +236,7 @@ func (o *ocsValidator) setStatusInsufficientResources(cpu int64, ram int64, disk
 		status = status + fmt.Sprint(TotalRAMGB, " RAM, excluding disk RAM resources ")
 	}
 	if disk < o.OCSRequiredDisk {
-		status = status + fmt.Sprint(o.OCSRequiredDisk, " Disks, ")
+		status = status + fmt.Sprint(o.OCSRequiredDisk, " Disks, of size aleast ", minDiskSize, " GB is required")
```

Comment:
I think that there is a spelling mistake: at least instead of aleast. I think that maybe the message should be: "minimum  minDiskSize GB is required".

Reply:
fixed